rust/datafusion/src/execution/context.rs

```diff
@@ -680,6 +680,15 @@ impl ExecutionConfig {
         self
     }
 
+    /// Replace the physical optimizer rules
+    pub fn with_physical_optimizer_rules(
```

Comment:
There is also `add_physical_optimizer_rule` that I added -> don't we want to use that?

Reply:
That won't let me remove rules :wink:

Reply:
The specific use case this time is that I want to remove the repartition rule because Ballista does not support that yet

Reply:
ah... makes sense :+1: